Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "PixieType[10]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[11]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[12]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[13]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[14]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[15]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[16]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[17]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[18]", "type": "ObjCoffeeCupSpawner::PixieType" },
+        { "name": "PixieType[19]", "type": "ObjCoffeeCupSpawner::PixieType" }
+      ]
+    },
+    "ObjGondolaBaseSpawner": {
+      "fields": [
+        { "name": "GondolaNum", "type": "uint32" },
+        { "name": "DirectionType", "type": "ObjGondolaBaseSpawner::DirectionType" },
+        { "name": "AngularSpeed", "type": "float32" },
+        { "name": "Radius", "type": "float32" },
+        { "name": "OffsetX", "type": "float32" },
+        { "name": "OffsetY", "type": "float32" },
+        { "name": "OffsetZ", "type": "float32" }
+      ]
+    },
+    "ObjGondolaSpawner": {
+      "parent": "ObjGondolaBaseSpawner",
+      "fields": [
+        { "name": "PixieType[0]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[1]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[2]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[3]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[4]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[5]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[6]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[7]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[8]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[9]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[10]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[11]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[12]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[13]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[14]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[15]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[16]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[17]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[18]", "type": "ObjGondolaSpawner::PixieType" },
+        { "name": "PixieType[19]", "type": "ObjGondolaSpawner::PixieType" }
+      ]
+    },
```

Comment:
Parameters named based on in-game functionality and parameters from other objects with similar functionality, specifically `enmMotora` for the naming of the 20 `PixieType` parameters. The type of the 20 parameters have been given the same type as to not clutter the enums data, considering they do the same.